src/core/i18n/utils.js

```diff
@@ -287,6 +295,29 @@ export function makeI18n(
       oneLineTranslationString(pluralKey), val);
   };
 
+  // Because the `addonType` we use internally is not translated, we needs this
+  // helper to return the translated names without effort.
+  i18n.addonType = function addonType(type) {
+    switch (type) {
+      case ADDON_TYPE_DICT:
+        return i18n.gettext('dictionary');
+      case ADDON_TYPE_LANG:
+        return i18n.gettext('language pack');
+      case ADDON_TYPE_OPENSEARCH:
+        return i18n.gettext('search plugin');
+      case ADDON_TYPE_THEME:
```

Comment:
I think you can just add `case ADDON_TYPE_COMPLETE_THEME:` here; as long as we "support" it we should return a value.